### ISSUE TITLE
process vs. processual entity

### ISSUE THREAD

**Issue by wdduncan** [obophenotype/uberon#2301]
@cmungall @matentzn @dosumis @addiehl @diatomsRcool @johnwjudkins

Ontobee shows [UBERON](http://www.ontobee.org/ontology/UBERON) has both `process` and `processual entity` classes. Is the `processual entity` intended to be the same as [BFO1.1 processual entity](http://www.ontobee.org/ontology/BFO11?iri=http://www.ifomis.org/bfo/1.1/span%23ProcessualEntity)?

Ontobee didn't provide a definition for `BFO1.1 processual entity`, but I seem to recall the definition being what is stated for `UBERON processual entity`. I'd have to do some digging to confirm/disconfirm.

 However, UBERON does have a `process` class that uses the [BFO process](http://www.ontobee.org/ontology/BFO?iri=http://purl.obolibrary.org/obo/BFO_0000015) IRI. So, this makes me wonder:
- What is the difference between `processual entity` and `process`? So, it is unclear why they are siblings.
- Is `processual entity` meant to be subclass of `process`?
- Is `process` meant to be a subclass of `processual entity`?

I think it would be great if this was made clear.  
Perhaps related to #1640

**Comment by shawntanzk** [comment]
added to agenda, will try to get an response to this

**Comment by bvarner-ebi** [comment]
@addiehl recommends that 'processual entity' should be merged with 'process' since they are likely representing the same thing and both may be present accidentally.

**Comment by wdduncan** [comment]
@bvarner-ebi Good. I agree with him. Somewhere else (not that it matters) I think I recommended the same as @addiehl

**Comment by shawntanzk** [comment]
As this is an upper ontology change - will need sign-off from @cmungall 
Will bring it to his attention :) thanks!

**Comment by cmungall** [comment]
Let's solve this one here: https://github.com/OBOFoundry/COB/issues/40

**Comment by wdduncan** [comment]
This ticket has been open for a while. Any plans to address it?

**Comment by dosumis** [comment]
Somehow missed this the first time around:

Background  - we wanted this for developmental stages - which need to allow for occurrent with arbitrary temporal boundaries.  In standard stage series divides development up into stages whose stage boundaries are defined by whatever morphological features are easy to score.   This is, I think, different from a process like glycolysis or signal transduction.

<img width="407" alt="image" src="https://github.com/obophenotype/uberon/assets/112839/6c819267-3c71-4263-ba8a-2556fa97c3c2">

**Comment by wdduncan** [comment]
[processual entity](https://ontobee.org/ontology/BFO11?iri=http://www.ifomis.org/bfo/1.1/span%23ProcessualEntity) is a BFO 1.1 concept, and in BFO 1.1 `process` is a child of `processual entity`.

<img width="258" alt="image" src="https://github.com/obophenotype/uberon/assets/3186638/afb7c934-eeaa-4d50-ba85-41863a1d9baa">

Mixing BFO 1.1 and BFO 2.0 concepts in this way introduces potential ambiguities in that it is unclear how to decide whether a class should be a `bfo2:process`, `bfo1.1:processual entity`, or (perhaps) `bfo1.1:process`.

At present [life cycle stage](http://purl.obolibrary.org/obo/UBERON_0000105) , is textually defined as being a kind of spatiotemporal region, which of course is not consistent to it being a child `processual entity`. 

As far as I can tell from the definition of [bfo2:process](https://ontobee.org/ontology/BFO?iri=http://purl.obolibrary.org/obo/BFO_0000015), `life cycle stage` can be a child of it, although linguistically it may sound a bit odd. Making such a change would address the potential ambiguities mentioned above.

**Comment by uberon** [comment]
Agreed. We need to be able to put stages in a different bucket than GO bps.
I guess this grouping will have to remain in uberon until
https://github.com/OBOFoundry/COB/issues/40 is resolved which has become
entangled with a lot of philosophical upper ontology discussion

**Comment by wdduncan** [comment]
> We need to be able to put stages in a different bucket than GO bps.

Yes, not every `process` is GO bps. Not sure why or if this precludes `life stage` from being a child of `bfo2:process`.

> I guess this grouping will have to remain in uberon until
[OBOFoundry/COB#40](https://github.com/OBOFoundry/COB/issues/40) is resolved which has become
entangled with a lot of philosophical upper ontology discussion

If you are waiting on the COB issue to resolve, I fear you will be waiting a very, very, long time. Better to move ahead on this now.

**Comment by wdduncan** [comment]
Any update on this?

**Comment by matentzn** [comment]
I am closing this issue in favour of https://github.com/OBOFoundry/COB/issues/40
